__tests__/compilers/compatability.test.tsx

```diff
@@ -332,6 +332,37 @@ This is an image: <img src="http://example.com/#\\>" >
     `);
   });
 
+
+  it('correctly parses and transforms image magic block with legacy data', () => {
+    const md = `
+[block:image]
+{
+  "images": [
+    {
+      "image": [
+        "https://files.readme.io/9ac3bf4-SAP_Folders_Note.png",
+        "SAP Folders Note.png",
+        806,
+        190,
+        "#e8e9ea"
+      ]
+    }
+  ]
+}
+[/block]
+## Tile View
+    `;
+
+    const tree = rdmd.mdast(md);
+    const rmdx = mdx(tree);
+    expect(rmdx).toMatchInlineSnapshot(`
+      "![806](https://files.readme.io/9ac3bf4-SAP_Folders_Note.png "SAP Folders Note.png")
```

Comment:
Hmmmm, why is the alt text being set to the `806` width/height dimension of the image here? That seems weird!

Reply:
The history of this is weird. I _think_ the old editor used to use a different magic block shape. It's from before my time, so doing the archaeology is harder for me. The JSON used to look something like:

```json
{
  "images": [{
    "image": [
      "path-to-some-image.jpg",
      "title",
      "300px",
      "center"
    ]
  }]
}
```

I don't think those docs were ever migrated? They've existed in the wild, but I don't ever remember seeing RDMD able to parse them correctly. The JSON shape I have seen, and is still in use:

```json
{
  "images": [{
    "image": [
      "path-to-some-image.jpg",
      "title"
    ],
    "sizing": "300px",
    "align": "center"
  }]
}
```

It comes up so rarely that I didn't bother adding a backwards-compatible parser?

Reply:
Ahhhh I see; that answers my question!